src/styles/blocks/catalog.scss

```
@@ -0,0 +1,28 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: (4 * $product-width) + (3 * $product-margin-right) + (2 * $padding-x);
+  margin: 0 auto;
+  padding: $padding-y $padding-x;
+
+  &__card {
+    box-sizing: border-box;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    padding: 32px 16px 16px;
+    width: $product-width;
+    margin-bottom: $product-margin-top;
+    margin-right: $product-margin-right;
+
+    background: white;
+  }
+
+  &__card:nth-child(4) {
+    margin-right: 0;
+  }
+
+  &__card:nth-child(8) {
+    margin-right: 0;
+  }
```

It is not very convenient to put 4, 8 and so on in brackets, what if you have 1000 products? it's way better to go like that: 
```suggestion
  &__card:nth-child(4n) {
    margin-right: 0;
  }
```